### PR TITLE
made changes to index and html files

### PR DESCRIPTION
**Pull request by Adeoye-J**
Made changes to the content

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.cop {
+    background-color: blue;
+}
```
Comment (TechWithJerry) [inline styles.css:3]: I don't like the class name, it is not descriptive. Make it "title"

File: index.html [hunk 2]
```diff
@@ -6,7 +6,7 @@
     <title>Github Usage Practice</title>
 </head>
 <body>
-    <h1>Lets have fun learning how to collaborate</h1>
+    <h1 class="cop">Lets have fun learning how to collaborate</h1>
```
Comment (TechWithJerry) [inline index.html:9]: make it reflect here too. Change to "title"

**Review by TechWithJerry — CHANGES_REQUESTED** [review]

**Review by Adeoye-J — COMMENTED** [review]
Changes made

File: styles.css (hunk 1 above)
Comment (TechWithJerry) [reply styles.css:3]: Alright, Good.

File: index.html [hunk 3]
```diff
@@ -6,7 +6,7 @@
     <title>Github Usage Practice</title>
 </head>
 <body>
-    <h1>Lets have fun learning how to collaborate</h1>
+    <h1 class="title">Lets have fun learning how to collaborate</h1>
```
Comment (TechWithJerry) [inline index.html:9]: Well-done

**Review by TechWithJerry — COMMENTED** [review]
Welldone. Completed. Keep up the good work.

**Review by TechWithJerry — APPROVED** [review]
Approved!

**Pull request merged**